contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -428,7 +426,6 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     }
     // Delete commitment
     delete s_requestCommitments[requestId];
-    emit RequestTimedOut(requestId);
```

Comment:
state changes here, some event should be emitted

Reply:
I see that you've moved the event into the Subscription, but seeing as you're ignoring the return value of this function there is no way to tell if this actually deleted anything.

Reply:
I'll keep it in both places

Reply:
Ended up adding an event here called `CommitmentDeleted` to be more consistent with the function name